client/src/main/java/org/spine3/protobuf/Durations2.java

```diff
@@ -82,14 +67,15 @@ public static Duration ofMinutes(long minutes) {
      * @param hours the number of hours, positive or negative
      * @return a non-null {@code Duration}
      */
-    public static Duration ofHours(long hours) {
-        final Duration duration = ofMinutes(safeMultiply(hours, MINUTES_PER_HOUR));
+    public static Duration fromHours(long hours) {
+        final Duration duration = fromMinutes(safeMultiply(hours, MINUTES_PER_HOUR));
         return duration;
     }
 
-    // Methods for brief computations with Durations like
-    //       add(hours(2), minutes(30));
-    /////////////////////////////////////////////////////
+    /*
+     * Methods for brief computations with Durations like
+     *       add(hours(2), minutes(30));
+     ******************************************************/
```

Comment:
We have `fromXxx` methods named after those available from Protobuf.

Our methods that do not have `from` prefix are for calculations similar to that mentioned in the comment. Mixing them with versions with `from` won't read well.